resip/recon/UserAgent.cxx

```diff
@@ -452,6 +459,23 @@ UserAgent::onSubscriptionNotify(SubscriptionHandle handle, const Data& notifyDat
    // Default implementation is to do nothing - application should override this
 }
 
+const char*
+UserAgent::sendMessage(const NameAddr& destination, const Data& msg, const Mime& mimeType)
+{
+   if(!mDum.getMasterProfile()->isMethodSupported(MESSAGE))
+   {
```

Comment:
but this function is for sending messages, it would get here when a MESSAGE is received?